App/Containers/Styles/RenameGroup.js

```diff
@@ -0,0 +1,43 @@
+import { StyleSheet } from 'react-native'
+
+import Colors from '../../Themes/Colors'
+import { color } from '../../styles'
+
+export default StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: Colors.modalColor,
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'stretch',
+    padding: 20
+  },
+  inputStyle: {
+    height: 60,
```

Comment:
For all the sizes, margins, paddings, etc, please use the const vales defined in the `styles` module. Similar to how you dealt with colors here.

Reply:
Same with font family.